src/Lumper.UI/ViewLocator.cs

```diff
@@ -6,12 +6,12 @@ namespace Lumper.UI;
 
 public class ViewLocator : IDataTemplate
 {
-    public static IControl Build(object? data)
+    public Control Build(object? param)
     {
-        if (data is null)
+        if (param is null)
             return new TextBlock { Text = "Null referenced object" };
 
-        var name = data.GetType().FullName!.Replace("ViewModel", "View");
```

Comment:
I think I just replaced this with the new ViewLocator implementation they give on their docs. I actually had to rewrite this class completely for trimmed builds anyway, but that's in a future commit. Since that's going to make review harder than anticipate I'm going to go through and try to `fixup` some later commit into these earlier ones so we have less cases like this.